find-channel/src/main/java/no/unit/nva/channel/ChannelRegistryClient.java

```diff
@@ -71,11 +75,18 @@ public ChannelRegistryClient(ObjectMapper objectMapper, CloseableHttpClient http
             HttpEntity entity = Optional.ofNullable(response.getEntity())
                     .orElseThrow(() -> new NoResultsFoundException(NO_RESULTS_FROM_SERVICE));
             String entityString = EntityUtils.toString(entity);
+            System.out.println("Response: " + entityString);
```

Comment:
Is this for logging?

Reply:
Yes, this is a legacy project, again the update task NP-1143